ports/espressif/boards/lilygo_ttgo_t8_s2_st7789/board.c

```diff
@@ -98,9 +98,9 @@ static void display_init(void) {
         bus,
         240,            // width (after rotation)
         135,            // height (after rotation)
-        52,             // column start
-        40,             // row start
-        90,             // rotation
+        40,             // column start
+        53,             // row start
+        0,              // rotation
```

Comment:
I tried these changes with a new firmware on my T8 ST7789 and it works as intended!

Reply:
Thanks!